recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.6.3" %}
+{% set version = "3.8.0" %}
 
 package:
   name: qgis
   version: {{ version }}
 
 source:
   url: https://qgis.org/downloads/qgis-{{ version }}.tar.bz2
-  sha256: b77790b47d978035e24d4652c5ad4227ae68205c056db54fa0fd2da6bf6f7b7d
+  md5: 81af2b33732ac4fa21ea2adf8bf72d4e
```

Comment:
Let's use the sha256 instead.

Reply:
The QGIS project provides the checksum in md5, not using it mean one need to download the package, check it with md5, and then create the sha256. that's why I changes it.

Reply:
The bot does all that for us. We just need to implement a way for the bot to update the branches.